src/features/commands.ts

```diff
@@ -78,26 +77,10 @@ export default function registerCommands(
         )
     );
 
-    disposable.add(
-        vscode.commands.registerCommand('csharp.showDecompilationTerms', async () =>
-            showDecompilationTerms(context, server)
-        )
-    );
-
     return new CompositeDisposable(disposable);
 }
 
-async function showDecompilationTerms(context: vscode.ExtensionContext, server: OmniSharpServer) {
-    // Reset the decompilation authorization so the user will be prompted on restart.
-    resetDecompilationAuthorization(context);
-
-    await restartOmniSharp(context, server);
-}
-
 async function restartOmniSharp(context: vscode.ExtensionContext, server: OmniSharpServer) {
-    // Update decompilation authorization.
-    server.decompilationAuthorized = await getDecompilationAuthorization(context);
```

Comment:
The acknowledgment was something we intended to remove to match the LspToolsHost behavior. Since it would be a lot to translate, I decided now was a good time to remove it.